src/font/system/std.rs

```diff
@@ -75,8 +75,7 @@ impl FontSystem {
             let now = std::time::Instant::now();
 
             //TODO only do this on demand!
-            for i in 0..db.faces().len() {
-                let id = db.faces()[i].id;
+            for id in db.faces().map(|face| face.id).collect::<Vec<_>>() {
```

Comment:
No need to collect into a temporary vec when you're only processing the `id`s one at a time and not storing this vec after? Same above.

Reply:
`db.faces()` returns an iterator now, because of the switch to `slotmap`. So iterating over `faces()` borrows `db`, but we also need to borrow `db` mutably inside the loop for the `db.make_shared_face_data(id)` call.
I also don't like this, do you have a better idea to solve this? I am also working on a PR that will remove the need for the loop, so hopefully this will only be a temporary issue.

Reply:
Hmm, that is quite ugly. Funnily enough we have a PR open to get values as a slice instead of iterator from `slotmap`, but that won't solve the borrow problem. Will look forward to that PR then :)

Reply:
Yeah, even I thought, after cherry-picking the commit from #89, "Do I really need this" and tried to remove it.